keps/sig-node/3675-resource-plugin-manager/README.md

```diff
@@ -546,9 +548,20 @@ following interface to manage resource sets.
         +RemoveResource(container)
         +AvailablResources(): resourceset
 
-3. Integration The Container Compute Interface “CCI” inside None and Static CPU Manager policies<br>
+3. Integration the Container Compute Interface “CCI” inside Cpu Manager Policies<br>
 The synchronization of cpuset state is done by a new policy component inside the 
-cpu manager which will be integrated inside the None and Static policies. For simplicity in alpha version we plan to support best-effort QoS for all Pods not requiring a resource management driver (equivalent to none policy). In later phases we plan to support Pods requiring the static policy without the need of resource driver (goal for Beta). The component acts as a bridge between CCI drivers and CPU manager. The drivers provide a decision (resource set stored in CCI Store) about a certain allocation and and cpu manager applies the given cpuset. Similar appoach could be followed with memory management.
+cpu manager which will cover both behaviours: none and static behaviour. The component acts also as
+a bridge between CCI drivers and CPU manager. The drivers provide a decision (resource set stored
+in CCI Store) about a certain allocation and cpu manager applies the given cpuset. 
+The component also ensures the mutual exclusive pod admission process for 
+applications requiring CCI drivers and applications without such requirements. 
+
+As the policy is new, existing policy behavior will not change and there will not be a need 
+to revalidate existing applications. For simplicity in alpha version we plan to support best-effort
+ QoS for all Pods not requiring a resource management driver (equivalent to none policy). 
+In later phases we plan to support Pods requiring the static policy capabilities. 
+Similar approach could be followed for the integration with memory management in Beta
+phase.
```

Comment:
I'm not sure this addresses earlier comment from @klueska . I *THINK* that his comment was more in the direction of keeping the flows (old managers/new CCI) as independent and as mutually exclusive as possible, thus avoiding the state sharing/sync entirely (Kevin please correct me here if I got you wrong)

Reply:
We followed that and updated the KEP accordingly.